src/framework/extensions/internal/extensioninstaller.cpp

```diff
@@ -36,11 +36,37 @@ Ret ExtensionInstaller::installExtension(const io::path_t srcPath)
         ExtensionsLoader loader;
         Manifest m = loader.parseManifest(data);
 
-        bool hasSame = provider()->manifest(m.uri).isValid();
-        if (hasSame) {
+        Manifest existingManifest = provider()->manifest(m.uri);
+        bool alreadyInstalled = existingManifest.isValid();
+        if (alreadyInstalled && existingManifest.version == m.version) {
             LOGI() << "already installed: " << m.uri;
+
+            interactive()->info("Info", "The extension is already installed.", { interactive()->buttonData(IInteractive::Button::Ok) });
```

Comment:
To make these strings translatable, use `trc` from `translation.h`, like this:
```suggestion
            interactive()->info(trc("extensions", "Info"), 
                                trc("extensions", "The extension is already installed."),
                                { interactive()->buttonData(IInteractive::Button::Ok) });
```

Also, we usually don't include the word "info" or "error", as that doesn't add much; so then you can do just
```suggestion
            interactive()->info(trc("extensions", "The extension is already installed."), 
                                { interactive()->buttonData(IInteractive::Button::Ok) });
```